shared/oval/sshd_limit_user_access.xml

```diff
@@ -0,0 +1,28 @@
+<def-group>
```

Comment:
This is in the wrong path. The new path for static OVAL files is `shared/templates/static/oval`.